modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoveryNetworkIssuesTest.java

```diff
@@ -243,14 +253,31 @@ public void testBackwardNodeCheckWithSameLoopbackSingleLocalAddress() throws Exc
      */
     @Test
     public void testBackwardNodeCheckWithSameLoopbackSeveralLocalAddresses() throws Exception {
-        doTestBackwardNodeCheckWithSameLoopback("0.0.0.0");
+        ListeningTestLogger testMethodLog = new ListeningTestLogger(log);
+
+        String startLogMsg = "Checking connection to node";
+
+        Collection<LogListener> lsnrs = new ArrayList<>();
+
+        lsnrs.add(LogListener.matches(startLogMsg).andMatches("result=success").times(1).build());
+        lsnrs.add(LogListener.matches(startLogMsg).andMatches("result=skipped").times(3).build());
```

Comment:
Why 3 skipped. It depends on the address actual number of the node 2. Consder mine detected set: /0:0:0:0:0:0:0:1%lo:47501, /127.0.0.1:47501, /2aa0:1370:8290:33c0:c023:152e:ac129:d2ec%stuid0f4:47501, /172.17.0.1:47501. Will faile with the timeout at the following `waitForCondition()`.